muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/communicator.js

```diff
@@ -418,7 +433,10 @@
             });
           }, this));
           
-          renderDustTemplate('communicator/communicator_items.dust', { items: data, sent: this._folderId == 'sent' }, $.proxy(function (text) {
+          renderDustTemplate('communicator/communicator_items.dust', {
+            items: data, sent: this._folderId == 'sent',
+            isStudent: $('.communicator').attr('data-student') == 'true' ? 1 : ''
```

Comment:
Dust (https://github.com/linkedin/dustjs/wiki/Dust-Tutorial#Logic_in_Templates)

"Take special care if you are trying to pass a boolean parameter. param=true and param=false do not pass true/false as you might expect. They are treated as references to variables named "true" and "false". Unlike JavaScript, they are not reserved names. Note that they are not reserved in JSON either so you can have a property named true or false. So you might think to pass 0 and 1 to your boolean-like parameter. That won't work either. dust's boolean testing ( {?xxx} is more of an existence test than a boolean test. Therefore, with param=1 and param=0 both value exists and so are considered true. Your best bet is to pass 1 and "", e.g. param=1 or param="". You could also leave off param="" if you are sure the name is not elsewhere in your JSON data and accessible."